knowledge-base/actions/datadog/labeler/action-security.yml

```diff
@@ -0,0 +1,11 @@
+name: "Pull Request Labeler"
+github-token:
+  action-input:
+    input: GITHUB_TOKEN
+    is-default: true
+  permissions:
```

Comment:
For this Action there are only:
`pull-request: read` permission `to get changed files from pr` view [here](https://github.com/DataDog/labeler/blob/1a043751b2faccb4f9078d542928e4c0d3f0e2e5/src/main.ts#L56), and 
`issues: write` permission `to add label to pr` view [here](https://github.com/DataDog/labeler/blob/1a043751b2faccb4f9078d542928e4c0d3f0e2e5/src/main.ts#L139)
is needed.

Also change the reference to [this](https://github.com/DataDog/labeler/blob/1a043751b2faccb4f9078d542928e4c0d3f0e2e5/src/main.ts#L8), where github token is utilized